app/overrides/add_deface_to_return_authorizations_index.rb

```diff
@@ -0,0 +1,27 @@
+Deface::Override.new(
```

Comment:
Are we sure that we have to create all these defaces and we can't do this using fewer defaces?

Deface takes a lot of time, so please ensure that we are using least number of defaces